src/wp-login.php

```diff
@@ -214,12 +214,12 @@ function login_header( $title = null, $message = '', $wp_error = null ) {
 	<?php
 	if ( 'confirm_admin_email' !== $action && ! empty( $title ) ) :
 		?>
-		<h1 class="screen-reader-text"><?php echo $title; ?></h1>
+		<h1 class="screen-reader-text"><?php echo esc_html( $title ); ?></h1>
 		<?php
 	endif;
 	?>
 	<div id="login">
-		<h1 role="presentation" class="wp-login-logo"><a href="<?php echo esc_url( $login_header_url ); ?>"><?php echo $login_header_text; ?></a></h1>
+		<h1 role="presentation" class="wp-login-logo"><a href="<?php echo esc_url( $login_header_url ); ?>"><?php echo esc_html( $login_header_text ); ?></a></h1>
```

Comment:
Maybe this could be even stricter, how about actually stripping any disallowed HTML here using `wp_kses()`, just allowing `<br>`, `<strong>` and similar uncritical tags.

Reply:
I think that should do fine. 
I considered `<span>`, but without class or style attributes that would do nothing, and allowing these would be wrong in my eyes. Maybe we go with your list and consider a filter at a later point?

Reply:
Hey @apermo ,
I've made the change as requested. 

I added a blank line after the closing PHP tag, before the next filter's docblock comment, to improve readability, as there was no blank line previously.